pkg/services/vulnerability.go

```diff
@@ -491,3 +491,12 @@ func (s *VulnerabilityService) GetWebVulnerabilitiesForService(ctx context.Conte
 
 	return vulns, nil
 }
+
+func (s *VulnerabilityService) GetWebVulnerabilityDetailsByID(ctx context.Context, vulnID uuid.UUID) (*domain.WebVulnerability, error) {
+	webVuln, err := s.vulnerRepo.GetWebVulnerabilityByID(ctx, vulnID)
+	if err != nil {
+		slog.Debug("Error when obtaining web vulnerability", slog.Any("vulnID", vulnID))
```

Comment:
`[Nit]` For error logging we should use `slog.Error`, and log the error for easier debugging!  
```suggestion
		slog.Error("Error when obtaining web vulnerability", 
		slog.Any("vulnID", vulnID),
		slog.Any("error", err),
		)
```

Reply:
You are right!, corrected